### ISSUE TITLE
JCenter deprecation

### ISSUE THREAD

**Issue by eclectice** [esoco/coroutines#8]
Is it possible for this library to be moved to the MavenCentral repo?

**Comment by jjzazuet** [comment]
@esocode a supplemental build of the library is now under my org's namespace:

https://mvnrepository.com/artifact/io.vacco.coroutines/coroutines
https://github.com/vaccovecrana/coroutines

I tried removing as much code as possible so that only the co-routines specific code and unit tests compile and run correctly, producing a 300Kb self contained jar for community use.

This should give you enough time to work on writing a minimal version of the core co-routines library, only bringing as little code as possible from the `objectrelations`, `esoco-monads`, `esoco-common` and omitting the `javax.validation` dependency completely (I found this while producing the minimal jar above).

The value of this library cannot be understated, and more projects should be using it, hence the need for a zero-dependencies, slim version.

Until then, folks can use the minimal jar above.

Let me know if I missed anything.

Thanks!
